recipes/corsikaio/meta.yaml

```diff
@@ -0,0 +1,52 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please remove all the comments from the example.

Reply:
done